WPMedia/engine-theme-sdk
# TMEDIA-300, ET AL image loading lazy cls fix

## Description
Use native image loading 

## Jira Ticket
- [TMEDIA-300](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-300&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria
reduce cls with top table xl 

## Test Steps
- git checkout image-native-lazy-load
- npx fusion start -f 
- Create top table page with 3 xl images 
- Run lighthouse and see cls 

## Effect Of Changes
### Before

![Screen Shot 2021-07-01 at 16 35 20](https://user-images.githubusercontent.com/5950956/124192337-bad81f00-da8a-11eb-8c29-9659964f9de1.png)

### After
![Screen Shot 2021-07-01 at 16 35 39](https://user-images.githubusercontent.com/5950956/124192319-b3b11100-da8a-11eb-9caf-363fad588870.png)

## Dependencies or Side Effects
- Lazy options is removed with a better functionality

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [] Confirmed this PR has reasonable code coverage
please explain why (so that we can fix it whenever it gets refactored).
